jib-core/src/main/java/com/google/cloud/tools/jib/frontend/CredentialRetrieverFactory.java

```diff
@@ -169,6 +170,9 @@ CredentialRetriever inferCredentialHelper(
               logger.info("  Caused by: " + ex.getCause().getMessage());
             }
           }
+
+        } catch (NonexistentServerUrlDockerCredentialHelperException | IOException ex) {
```

Comment:
Should `NonexistentServerUrlDockerCredentialHelperException` and `NonexistentDockerCredentialHelperException ` be subclasses of `CredentialRetrievalException`?

Reply:
That will potentially be done as part of #832